server/src/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -3085,9 +3085,10 @@ public GuestVlan dedicateGuestVlanRange(DedicateGuestVlanRangeCmd cmd) {
         // Verify account is valid
         Account vlanOwner = null;
         if (projectId != null) {
-            if (accountName != null) {
-                throw new InvalidParameterValueException("accountName and projectId are mutually exclusive");
-            }
+            //accountName and projectId are mutually exclusive
```

Comment:
@nitin-maharana I disagree this change. We'd better keep like it.
I suggest to make ui change to add a new field 'scope' which can be set to 'domain', 'account', 'project'.
It is similar to the scope field in the dialog to create a shared network in advanced zone.

scope='Domain', domain field is shown.
scope='Account, domain and account are shown
scope='Project', domain and project are shown.